src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,63 @@
+/// <reference path="../../../types/react-tether/index.d.ts" /> Required to make dts-generator bundle react-tether definition file.
+
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  constantClasses?: string;
+  displayOnShow?: string;
+}
+
+export interface IUserFeedbackStyle {
+  display: string;
+  className: string;
+}
+
+export const userFeedbackState = {
+  VALID: 'VALID',
+  WARNING: 'WARNING',
+  ERROR: 'ERROR',
+};
+
+export class UserFeedback extends React.Component<IUserFeedbackProps, any> {
+
+  private getDefaultUserFeedbackStyle(): IUserFeedbackStyle {
+    return {
+      display: 'none',
```

Comment:
You mean like toggling a 'hidden' class like the one we already have? The advantage I see with inline style for the display is that you can choose which type of display you want (inline-block, block, etc.). If you use a class, you are either forced to choose one display type for your class, or you'll have to create a class for each display type. In one way I see it as a bit limiting, and on the other as a bit redundant. What are your thoughts?